src/dndbeyond/content-scripts/encounter.js

```diff
@@ -22,6 +25,23 @@ function documentModified(mutations, observer) {
     character.parseStatBlock(monster);
 }
 
+function updateCombatTracker() {
+    if ($(".turn-controls__next-turn-button").length) {
+        $(".ui-dialog-buttonpane").hide(); // get rid of turn controls, to make it more obvious to do things from DDB.
+        const combat = Array.from($(".combatant-card.in-combat")).map(x => [
+            $(".combatant-summary__name", x).text(),
+            $(".combatant-card__initiative-value", x).text(),
+            $(x).hasClass("is-turn"),
+        ]);
+        const req = {
+            action: "combat-tracker",
+            combat,
+        };
+        console.log("Sending combat update", combat);
```

Comment:
I would name the action "update-combat" to be consistent with the other actions, and would only send the message if the setting is enabled (i.e: check the setting here rather than in the VTT)

Reply:
Oops, I deleted the message here.

Even renamed to update-combat, and that is already done in the if statement protecting whether this function gets called.